src/Elrond/Codec.cpp

```diff
@@ -25,7 +25,6 @@ std::string Codec::encodeBigInt(const std::string& value) {
     return encodeBigInt(uint256_t(value));
 }
 
-// For reference, see https://docs.elrond.com/developers/developer-reference/elrond-serialization-format/#arbitrary-width-big-numbers.
```

Comment:
Can we just keep this comment for now?

Reply:
Added it. Note: `docs.multiversx.com` will become available in ~24-48 hours.